crates/red_knot_python_semantic/resources/mdtest/import/conflicts.md

```diff
@@ -0,0 +1,63 @@
+# Conflicting attributes and submodules
+
+## Via import
+
+```py
+import a.b
+
+reveal_type(a.b)  # revealed: <module 'a.b'>
+```
+
+```py path=a/__init__.py
+b = 42
+```
+
+```py path=a/b.py
+```
+
+## Via from/import
+
+```py
+from a import b
+
+reveal_type(b)  # revealed: Literal[42]
+```
+
+```py path=a/__init__.py
+b = 42
+```
+
+```py path=a/b.py
+```
+
+## Via both
+
+```py
+import a.b
+from a import b
+
+reveal_type(b)  # revealed: <module 'a.b'>
```

Comment:
It is intentional to test the type of `b` here. In this case, because `import a.b` occurs first, what we are testing here does match the runtime behavior.

I guess it would be fine to _also_ test the type of `a.b`.

Reply:
Added a test of `a.b` as well